css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
make sure to properly adjust the order of these selectors to match the instruction order. It demonstrates specificity much better that way!